0-bootstrap/modules/jenkins/scripts/jenkins_gce_startup_script.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+echo "**** Startup Step 1/9: Update apt-get repositories. ****"
+sudo apt-get update
+
+echo "**** Startup Step 2/9: Install Java. ****"
+sudo apt-get install -y default-jdk
+
+echo "**** Startup Step 3/9: Set variables. ****"
+export TERRAFORM_DIR="/opt/terraform/"
+export TERRAFORM_VERSION="0.12.24"
+
+echo "**** Startup Step 4/9: Create tahe directory for Terraform in /opt. ****"
+mkdir $TERRAFORM_DIR && cd $TERRAFORM_DIR || exit
```

Comment:
I am not sure why we have exit here? A startup script would be running as root  and would have required permissions?

Reply:
It shows a lint error. If the mkdir doesn't work for any reason, the script is cancelled

Reply:
even with `mkdir -p`?

Reply:
Actually, the lint message is related to the `cd` command - see https://github.com/koalaman/shellcheck/wiki/SC2164

So I understand we should leave it there.

Reply:
`set -e` might be cleaner as it makes sense to exist if any of these steps fail not just the `cd` step.